### PR TITLE
Added Fibonacci Numbers

### PR DESCRIPTION
**Pull request by LashaBuxo**

### CODE REVIEW

File: Basic Math/Fibonacci_Numbers.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Calculate matrix^8
+| 34 21 |
+| 21 13 |
+```
+### Step 3
+F(8)=21 
+
+## Implementation
+
+-   [C++](https://www.tutorialspoint.com/cplusplus-program-to-find-fibonacci-numbers-using-matrix-exponentiation)
```
Comment (AnupKumarPanwar) [inline Basic Math/Fibonacci_Numbers.md:61]: Can you check other "The Algorithms" repos for implementation and user those links . I'm sure you'll find them in python, java and c++ repos.

File: Basic Math/Fibonacci_Numbers.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+`[0, 1, 1, 2, 3, 5, 8, 13, 21, 34,  ...]`
+
+
+## Applications
+
+Finding 
+```N-th``` member of this sequence would be useful in many Applications:
+
+-   Recently Fibonacci sequence and golden ratio are of great interest to the researchers in many fields of
```
Comment (vbrazo) [inline Basic Math/Fibonacci_Numbers.md:13]:
```suggestion
-   Recently Fibonacci sequence and the golden ratio are of great interest to researchers in many fields of
```

File: Basic Math/Fibonacci_Numbers.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Calculating Fibonacci numbers
+
+In mathematics, the Fibonacci numbers, commonly denoted F(n), form a sequence, called the Fibonacci sequence, such that each number is the sum of the two preceding ones, starting from 0 and 1.  The Sequence looks like this:
```
Comment (vbrazo) [inline Basic Math/Fibonacci_Numbers.md:3]:
```suggestion
In mathematics, the Fibonacci numbers commonly denoted F(n), form a sequence, called the Fibonacci sequence, such that each number is the sum of the two preceding ones, starting from 0 and 1.  The Sequence looks like this:
```

File: Basic Math/Fibonacci_Numbers.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+-   Recently Fibonacci sequence and golden ratio are of great interest to the researchers in many fields of
+science including high energy physics, quantum mechanics, Cryptography and Coding. 
+ 
+
+## Steps
+
+1.  Prepare Base Matrice
+2.  Calculate the power of this Matrice
+3.  Take Corresponding value from Matrixe
```
Comment (vbrazo) [inline Basic Math/Fibonacci_Numbers.md:21]:
```suggestion
3.  Take Corresponding value from Matrix
```

**Review by vbrazo — APPROVED** [review]
LGTM

**Pull request merged**